tests/client/core/api/test_api.js

```diff
@@ -138,7 +138,8 @@ describe('api', () => {
         })
         .once()
         .returns(mockResponse());
-      return api.login({api: {}, code: 'my-code', state: 'my-state'}).then((apiResponse) => {
+      const lang = 'en-US';
+      return api.login({api: {lang}, code: 'my-code', state: 'my-state'}).then((apiResponse) => {
```

Comment:
Could take the opportunity to move the `then` down onto the next line for all of these.